scripts/scenarios/cloud7/cloud7-4nodes-default.yml

```diff
@@ -0,0 +1,232 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: sbd
+      sbd:
+        nodes:
+          @@controller1@@:
+            devices:
+            - "##sbd_device_services_controller1##"
+          @@controller2@@:
+            devices:
+            - "##sbd_device_services_controller2##"
+        watchdog_module: softdog
+      per_node:
+        nodes:
+          @@controller1@@:
+            params: ''
+          @@controller2@@:
+            params: ''
+  deployment:
+    elements:
+      pacemaker-cluster-member:
+      - @@controller1@@
+      - @@controller2@@
+      hawk-server:
+      - @@controller1@@
+      - @@controller2@@
+- barclamp: database
+  attributes:
+    sql_engine: postgresql
```

Comment:
Mariadb is the default sql engine for soc7, isn't it? Or do we have another job for mariadb?